src/wallet/walletutil.cpp

```diff
@@ -54,8 +54,9 @@ std::vector<fs::path> ListWalletDir()
     const fs::path wallet_dir = GetWalletDir();
     const size_t offset = wallet_dir.string().size() + 1;
     std::vector<fs::path> paths;
+    boost::system::error_code ec;
```

Comment:
Might be a good place to comment that errors are ignored.

Reply:
Good point, will add.

Reply:
Please don't ignore error, at least log them